packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,240 +7,20 @@ section: get-started
 import './get-started.css';
 import { Divider } from '@patternfly/react-core';
 
-## PatternFly 5.1 
+## PatternFly 6 alpha
 
-As part of our [quarterly release cadence](/get-started/about#patternfly-release-cadence), we're excited to announce that PatternFly 5.1 is now live.
+We're excited to announce that our alpha release of PatternFly 6 is now live. Here are the highlights that you should know about!
 
-To help provide a high-level view of what's new, we've pulled together the major highlights from this release. Keep reading to learn more!
+### Penta
 
-### Promoted package versions
+As you've probably already noticed on the alpha website: PatternFly has a new look. Our brand new visual theme is called "Penta". Its visual style is more modern and refined, so that you can create UIs that are more seamless and lightweight.
```

Comment:
```suggestion
As you've probably already noticed on the alpha website: PatternFly 6 has a new look. Our brand new visual theme is more modern and refined, so that you can create UIs that are more seamless and lightweight.
```